hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationManager.java

```diff
@@ -1055,6 +1190,13 @@ public void run() {
             }
         }
 
+        /**
+         * Called on the master node. It will send promotions to the destinations and commit if the destinations successfully
```

Comment:
Yeah, it actually does the same thing, the difference being that `promoteBackupsForMissingOwners` accepts a `Map<Address, Collection<MigrationInfo>> promotions` while the `commitPromotionMigrations` accepts a single entry of that map, being `Address destination, Collection<MigrationInfo> migrations`.
The name is probably wrong since it does not reflect what it does but rather the context in which it is used (when promoting backups for missing owners).